src/frontend/src/components/FilterPanel/components/GenomeRecoveryFilter/index.tsx

```diff
@@ -12,6 +12,15 @@ interface Props {
   updateGenomeRecoveryFilter: (selected?: string) => void;
 }
 
+const GENOME_RECOVERY_OPTIONS = [
```

Comment:
This is kind of an interesting one. The `MenuSelect` component `value` wasn't able to identify that a previously selected option was previously selected because it did comparison with `===`, and since the OPTIONS were created inside component render, it was a new batch of objects each time, so they failed equality. Just pushing it outside of component render so it only gets made once is enough to get the value/option equality checking working so MenuSelect can now if it's looking at something it has already selected.

Reply:
cool glad it was a relatively easy fix